src/Button/Button.component.tsx

```diff
@@ -502,8 +482,13 @@ export const Button = ({
     minWidth,
     prefix,
     suffix,
+    onMouseDown,
+    onMouseUp,
```

Comment:
This probably isn't an issue, but would we want onTouchStart/End to mirror this behavior?